common/src/main/java/org/opensearch/ml/common/breaker/BreakerName.java

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.breaker;
+
+public enum BreakerName {
+    MEMORY
```

Comment:
May I ask if any breakers based on "CPU", "garbage collection" or "Disk usage" will be added?

Reply:
Yeah, that's possible. We can add more types later.